experimental/LieAlgebras/test/CartanMatrix-test.jl

```diff
@@ -1,27 +1,36 @@
 @testset "LieAlgebras.CartanMatrix" begin
-  @testset "cartan_matrix(fam::Symbol, rk::Int)" begin
+  @testset "cartan_matrix" begin
+    for i in -1:10
+      if i < 1
+        @test_throws ArgumentError cartan_matrix(:A, i)
+      end
+      if i < 2
+        @test_throws ArgumentError cartan_matrix(:B, i)
+      end
+      if i < 2
+        @test_throws ArgumentError cartan_matrix(:C, i)
+      end
+      if i < 4
+        @test_throws ArgumentError cartan_matrix(:D, i)
+      end
+      if i < 6 || i > 8
+        @test_throws ArgumentError cartan_matrix(:E, i)
+      end
+      if i != 4
+        @test_throws ArgumentError cartan_matrix(:F, i)
+      end
+      if i != 2
+        @test_throws ArgumentError cartan_matrix(:G, i)
+      end
+    end
```

Comment:
can you transform this block of tests into tests for `is_cartan_type` and then test the cartan matrix constructor via
```julia
for i in -1:10, fam in (:A, :B, :C, :D, :E, :F, :G)
  if is_cartan_type(fam, i)
    cartan_matrix(fam, i)
  else
    @test_throws ArgumentError cartan_matrix(fam, i)
  end
end
```
?